azkaban-web-server/src/main/bash/azkaban-web-shutdown.sh

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/env bash
 # Shutdown script for azkaban web server
-installdir="$(dirname $0)/.."
+set -o nounset
+source "$(dirname $0)/util.sh"
 
+installdir="$(dirname $0)/.."
+maxattempt=5
```

Comment:
I will reduce maxattempt to 3, so that it can fail faster. But I don't think we should spend too much effort tweaking these numbers as long as they are reasonable.